src/lib/components/ProductForm.svelte

```diff
@@ -439,6 +441,42 @@
 			{/each}
 		{/if}
 
+		<label class="checkbox-label">
+			<input
+				class="form-checkbox"
+				type="checkbox"
+				bind:checked={product.hasVariations}
+				name="hasVariations"
+				disabled={!product.standalone}
+			/>
+			Product has light variations (no stock nor price difference)
+		</label>
+		{#if product.hasVariations}
+			{#each [...(product.variations || []), ...Array(variationLines).fill( { name: '', value: '' } )].slice(0, variationLines) as variation, i}
+				<div class="flex gap-4">
+					<label class="form-label">
+						Name
+						<input
+							type="text"
+							name="variations[{i}].name"
+							class="form-input"
+							value={variation.name}
+						/>
+					</label>
+					<label class="form-label">
+						Value <input
+							type="text"
+							name="variations[{i}].value"
```

Comment:
Those inputs should be on `variationLabels`.

There should be also `variations` but as hidden inputs - which are "slugified" for each name/value